spec/controllers/gifts_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe GiftsController do
+  include LoginMacros
+  include RedirectExpectationHelper
+
+  describe "toggle_rejected" do
+    let (:gift) { create(:gift) }
+
+    it "errors and redirects to login page if no user is logged on" do
+      pending("AO3-5445 fix")
```

Comment:
We use `xit` for [pending specs ](https://github.com/otwcode/otwarchive/wiki/Automated-Testing#Pending_Tests)